virttest/storage.py

```diff
@@ -536,7 +624,11 @@ def get_backup_set(filename, backup_dir, action, good):
         backup_dir = params.get("backup_dir", "")
         if not os.path.isabs(backup_dir):
             backup_dir = os.path.join(root_dir, backup_dir)
-        if params.get('image_raw_device') == 'yes':
+        if self.is_remote_image():
+            backup_set = get_backup_set(self.image_filename, backup_dir,
+                                        action, good)
+            backup_func = self.copy_data_remote
```

Comment:
Seems we need the same thing for `save_image`

Reply:
Done